src/System.Memory/tests/Span/Clear.cs

```diff
@@ -198,6 +199,31 @@ public static void ClearReferenceTypeLonger()
             Assert.Equal<string>(expected, actual);
         }
 
+        [Fact]
+        public static void ClearReferenceTypeSlice()
```

Comment:
Can you also add such a test for non-reference types?

Reply:
This commit isn't the appropriate place for such a test. That test should be added when (if) we change the blittable T implementation.